src/main/resources/db/migration/V184__add_courttype_search_name.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE public.search_courttype
```

Comment:
Would it be easier to search by the name rather than a new column? Is there a reason for this? 

I suppose either way we would need to make sure its documented in terms of what inputs are allowed for the endpoint

Reply:
to make is easier to search  for example http://localhost:8080/courts/court-types/county,crown,magistrates,family,tribunal as opposed to http://localhost:8080/courts/court-types/county court ,crown court ,magistrates court ,family court ,tribunal